### ISSUE TITLE
Preview fails if no Config Module defined

### ISSUE THREAD

**Issue by SorraTheOrc** [Avindr/MxM-IssueTracking#121]
Attempting to preview an animation in a module will fail silently if no config module is defined.

Repro:

![image](https://user-images.githubusercontent.com/250240/116897641-5a437780-abea-11eb-8565-0cf049cb4571.png)

In the above setup I have already added events to the Melee Punch Attacks. This was done prior to creating the module and thus a config was present. If I click on preview on one of those animations it works as expected. However:

1. Do NOT preview one of the already configure punch attacks 
2. Click the preview button when setup as above
3. No errors, no preview

OR

1. Preview one of the already configured punch attacks
2. Works as expected
3. Click preview for one of the movement animations (no tracks added yet)
4. Timeline shows only preview button and errors in console:
![image](https://user-images.githubusercontent.com/250240/116898107-d8a01980-abea-11eb-9969-5ab40afcb347.png)

Adding a config module resolves the problem. Maybe simply add a check for the config module and display a dialog if one does not exist.

Unity Version: 2019.4.21f1
MxM Version: 2.2.9

**Comment by Avindr** [comment]
Thanks, I'll fix this for the next build.

**Comment by SorraTheOrc** [comment]
I found another oddity which I think may be related. I'm not filing a separate bug as I've not confirmed it is even a bug at all. But I feel it may be related to this...

I originally created a handful of attack animations with events as collections in the pre processor.
I exported those into modules and switched to using an entirely module driven flow
This resulted in the error above.

The new issue I have found is that when I added a new attack animation to the module my existing animations lost the reference to the Event ID. The event itself and all timing information remained, only the event ID was lost (the dropdown in the timeline was empty).

Now that I have reset the event ID I have not observed this error again.
